src/Microsoft.AspNet.Mvc.Core/ControllerActionInvokerProvider.cs

```diff
@@ -67,7 +69,8 @@ public void OnProvidersExecuting([NotNull] ActionInvokerProviderContext context)
                                     _modelValidatorProviders,
                                     _valueProviderFactories,
                                     _actionBindingContextAccessor,
-                                    _tempData);
+                                    _tempData,
+                                    _maxAllowedErrorsInModelState);
```

Comment:
This is different from what we'd discussed - but you could assign the value here rather than flow it two additional classes: `context.ActionContext.ModelStateDictionary = _maxModelValidationErrors;;`.It's less obvious, but it's fewer lines of code change. @rynowak,  thoughts?